gpii/node_modules/lifecycleManager/test/js/LifecycleManagerQueueTests.js

```diff
@@ -60,7 +60,7 @@ https://github.com/GPII/universal/blob/master/LICENSE.txt
 
     // ensure that we're processing queue in the correct order:
     jqUnit.asyncTest("Tasks are being processed on addition", function () {
-        var lifecycleManager = gpii.lifecycleManager();
+        var lifecycleManager = gpii.lifecycleManager(gpii.tests.lifecycleManager.privateMatchMakerOptions);
```

Comment:
Much cleaner than always defining this as a subcomponent of the lifecycleManager and then always configuring it away, seems to be to just configure the subcomponent at the point of definition of the lifecycleManager within the FlowManager. As far as I can see, the only dangerous external dependency is on the deviceReporter?